health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/MapViewPopup.js

```
@@ -0,0 +1,325 @@
+};
+
+const ViewMap = ({ lat, long, type }) => {
+  const { t } = useTranslation();
+  const [baseMaps, setBaseMaps] = useState({});
+  const [showBaseMapSelector, setShowBaseMapSelector] = useState(false);
+  const [selectedBaseMap, setSelectedBaseMap] = useState({});
+  const [selectedBaseMapName, setSelectedBaseMapName] = useState("");
+  const basemapRef = useRef();
+  const mapRef = useRef(null);
+  const markerRef = useRef(null);
+  var [map, setMap] = useState(null);
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)